### DISCUSSION TITLE
After Last update lost access to mail panel

### DISCUSSION THREAD

**Discussion by HomaEEE** [filamentphp/filament#7958] [Help]
<img width="571" alt="Снимок экрана 2023-08-21 в 16 11 44" src="https://github.com/filamentphp/filament/assets/1429343/30743e49-7088-4d4f-81c3-d4a1ebbb17ab">

V.3.0.29

Don't access to route of Filament PHP

**Comment by leandrocfe** [comment]
run `composer update` again
